examples/auth/main.go

```diff
@@ -14,31 +14,47 @@ func main() {
 		ctx,
 		qdb.WithAddress("localhost:9009"),
 		qdb.WithAuth(
-			"testUser1",
-			"5UjEMuA0Pj5pjK8a-fa24dyIf-Es5mYny3oE_Wmus48"),
+			"testUser1", // token name here
+			"5UjEMuA0Pj5pjK8a-fa24dyIf-Es5mYny3oE_Wmus48", // token here
+		),
 	)
 	if err != nil {
 		log.Fatal(err)
 	}
 	// Make sure to close the sender on exit to release resources.
 	defer sender.Close()
+
 	// Send a few ILP messages.
+	bday, err := time.Parse(time.DateOnly, "1856-07-10")
+	if err != nil {
+		log.Fatal(err)
+	}
 	err = sender.
-		Table("trades").
-		Symbol("name", "test_ilp1").
-		Float64Column("value", 12.4).
-		At(ctx, time.Now().UnixNano())
+		Table("inventors").
+		Symbol("born", "Austrian Empire").
+		TimestampColumn("birthdate", bday). // Epoch in micros.
+		Int64Column("id", 0).
+		StringColumn("name", "Nicola Tesla").
+		At(ctx, time.Now()) // Epoch in nanos.
```

Comment:
is `time.Now()` explicit nanos?

Reply:
`time.Now()` returns https://pkg.go.dev/time#Time